src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationlocation/Example1.java

```diff
@@ -10,18 +10,29 @@
 
 package com.puppycrawl.tools.checkstyle.checks.annotation.annotationlocation;
 
+import org.antlr.v4.runtime.misc.NotNull;
+import org.mockito.Mock;
+
+interface DataLoader {
+
+}
+
+
 // xdoc section -- start
 class Example1 {
-  @NotNull private boolean field1; //ok
+  @NotNull
+  private boolean field1; //ok
   @Override public int hashCode() { return 1; } //ok
   @NotNull //ok
   private boolean field2;
   @Override //ok
   public boolean equals(Object obj) { return true; }
-  @Mock
-  DataLoader loader; //ok
-  @SuppressWarnings("deprecation") DataLoader loader; //violation
-  @SuppressWarnings("deprecation") public int foo() { return 1; } //violation
-  @NotNull @Mock DataLoader loader; //violation
+  @Mock DataLoader loader1; //ok
+  @SuppressWarnings("deprecation") DataLoader loader2;
+  //violation above, 'Annotation 'SuppressWarnings' should be alone on line'
```

Comment:
Please do:
1. `//violation` -> `// violation`
2. Open ticket to create checkstyle module for inputs to enforce this.

Reply:
1. `//violation` -> `// violation.` done. 
2. Open ticket to create checkstyle module for inputs to enforce this.
     I have written an issue. Please let me know if it needs any alteration. 
     https://github.com/checkstyle/checkstyle/issues/13797